### PR TITLE
Apis

### PR DESCRIPTION
**Pull request by deependra-default**

### CODE REVIEW

File: accounts/models.py [hunk 1]
```diff
@@ -26,10 +26,10 @@ class Meta:
         verbose_name_plural =_('users')
     
     def __str__(self):
-        return self.email
+        return f'{self.first_name} {self.last_name}'
 
     def get_full_name(self):
-        full_name = f'{self.first_name}  {self.last_name}'
+        full_name = f'{self.first_name} {self.last_name}'
         return full_name.strip()
```
Comment (evilcudmovement) [inline accounts/models.py:33]: Why use .strip()?

Why is it required?

File: banking/management/commands/load_test_data.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+from banking.models import *
+from datetime import datetime
+from django.core.management.base import BaseCommand
+from decimal import Decimal
+from faker import Faker
+fake = Faker()
+
+def load_data():
+    user = User.objects.create(
+    username=fake.user_name(),
```
Comment (evilcudmovement) [inline banking/management/commands/load_test_data.py:11]: Please indent properly

File: banking/management/commands/load_test_data.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    email=fake.email(), 
+    user_type='CU',
```
Comment (evilcudmovement) [inline banking/management/commands/load_test_data.py:13]: 'CU' should not be hard coded. Instead import from some common file defining constants/types both here and inside the models file

File: banking/management/commands/load_test_data.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
+from accounts.models import User
+from banking.models import *
+from datetime import datetime
+from django.core.management.base import BaseCommand
+from decimal import Decimal
+from faker import Faker
+fake = Faker()
+
+def load_data():
+    user = User.objects.create(
```
Comment (evilcudmovement) [inline banking/management/commands/load_test_data.py:10]: Are these fixtures? If these are not fixtures then please use Factory pattern, don't create these  objects for real in the db.

File: banking/permissions.py [hunk 5]
```diff
@@ -0,0 +1,52 @@
+from rest_framework import permissions
+from django.http import HttpResponseForbidden
+from django.core.exceptions import PermissionDenied
+
+
+class IsEmployee(permissions.BasePermission):
```
Comment (evilcudmovement) [inline banking/permissions.py:6]: Do we need to define custom classes for this? Can't this be done using django roles?

File: banking/permissions.py [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+    """
+    """
+    def has_permission(self, request, view):
+        """
+        Should simply return, or raise a 403 response.
+        """
+        if not request.user.is_authenticated:
```
Comment (evilcudmovement) [inline banking/permissions.py:13]: Why check for authentication here?

File: banking/views.py [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+                transact.transaction_method = data["transaction_method"]
+                transact.transaction_type = data["transaction_type"]
+                transact.amount = data["amount"]
+                transact.account_id = data["account"]
+                transact.save()
+
+            except Exception as e:
+                return Response({"status": "Unable to create Credit transaction"})
+
+            # TODO will place is some where or i'll create model function for that email will send to account holder from bank
```
Comment (evilcudmovement) [inline banking/views.py:48]: TODO: Comment is not clear, can't understand what it means

File: banking/views.py [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+                    "Description",
+                    "Reference Number",
+                ]
+            ]
+            transact = Transaction.objects.filter(
+                account__account_number__in=data["acc_ids"],
+                created_at__gte=data["start_date"],
+                created_at__lte=data["end_date"],
+            )
+            for ts in transact:
```
Comment (evilcudmovement) [inline banking/views.py:113]: Instead build this into a celery task that sends an email with the csv. Let's not respond with csv in response

File: banking/permissions.py (hunk 5 above)
Comment (deependra-default) [reply banking/permissions.py:6]: I'm looking this will do it later

**Pull request merged**